src/clib/bget.c

```diff
@@ -535,9 +535,9 @@ static long numdget = 0, numdrel = 0; /* Number of direct gets and rels */
 
 /* Automatic expansion block management functions */
 
-static int (*compfcn) _((bufsize sizereq, int sequence)) = NULL;
-static void *(*acqfcn) _((bufsize size)) = NULL;
-static void (*relfcn) _((void *buf)) = NULL;
+static int (*compfcn)(bufsize sizereq, int sequence) = NULL;
```

Comment:
I believe we are going to use malloc everywhere, and get rid of bget. @jedwards4b am I correct in that?

Reply:
I recently saw some errors in which a model code did not write correct and complete output files when using malloc, switching back to bget solved the problem.  I need to investigate further before I will be ready to remove the bget code - but that is the plan in the long run.

Reply:
OK, thanks for the update!